The list of additional fixes specific to this version of mutt.

This patch will not be sent upstream.

diff --git a/PATCHES b/PATCHES
index e69de29..1350eab 100644
--- a/PATCHES
+++ b/PATCHES
@@ -0,0 +1,3 @@
+In addition, these upstream defects are fixed:
+#2064 wish: imap operations should be interruptable
+#3392 Whitespace garbage in pager mode
